### ISSUE TITLE
Source not apparent from display

### ISSUE THREAD

**Issue by pitman** [okfn/bibserver#48]
From the display

http://bibsoup.net/collection/hartley

it is not apparent from what url the data was uploaded. The url, in this instance http://rsise.anu.edu.au/~hartley/hartley.bib
should be apparent, as it is in 

http://bibserver.berkeley.edu/cgi-bin/bibs7?source=http://rsise.anu.edu.au/~hartley/hartley.bib

See text "Display created by BibServer from this bibtex source file http://rsise.anu.edu.au/~hartley/hartley.bib" at bottom of page
with hyperlink to the source file.

Providers of source data should be encouraged to provide metadata about their collection which can then be made part of the default BibServer display of that collection.  Formatting and handling of that metadata is another issue. First lets ensure that the
use case with no metadata besides a source url is accomodated.
Allowing uploaders to name their collection uploads complicates the issue raised here.  The legacy BibServer names its caches by  source url, so the issue of what happens if the same url is uploaded with two different collection names does not arise.  It does not seem a good idea to be making copies of the data from the same url even with different collection names, but this may require further discussion.

To summarize. 
- There should be just one copy of data from a given url in the BibSoup. 
- This entire copy should be overwritten by any re-upload of that dataset by any user.  
- The source url should always be apparent from the BibSoup display.
- There should be a button on the display which allows refreshing of the dataset.
  These are well tested and successful features of the legacy BibServer display. Lets demonstrate these features before
  trying more complex things.
